mne/realtime/epochs.py

```diff
@@ -326,16 +327,24 @@ def _process_raw_buffer(self, raw_buffer):
 
         # detect events
         data = np.abs(raw_buffer[self._stim_picks]).astype(np.int)
-        data = np.atleast_2d(data)
-        buff_events = _find_events(data, self._first_samp, verbose=verbose,
-                                   **self._find_events_kwargs)
-
+        #if there is a previous buffer check the last samples from it too        
+        if self._last_buffer != None:        
```

Comment:
you want to use `is not None`. See: http://docs.quantifiedcode.com/python-code-patterns/readability/comparison_to_none.html